src/stan/lang/grammars/statement_grammar_def.hpp

```diff
@@ -328,7 +340,10 @@ namespace stan {
         %= var_r(_r1)
         >> indexes_g(_r1)
         >> assignment_operator_r
-        >> (eps > expression_rhs_r(_r1))
+        >> (eps[validate_lhs_var_assgn_f(_val, _r1, _pass,
```

Comment:
Why is this not a single `>`?

Reply:
it's because we're using Boost Fusion to extract attributes of this rule  (series of `BOOST_FUSION_ADAPT_STRUCT` declarations at beginning of file.  must use `>>` to set up the sequence of rule attributes which are the elements of the struct.